low-level/sspl-ll.spec

```diff
@@ -112,12 +109,6 @@ SSPL_DIR=/opt/seagate/%{product_family}/sspl
 # [ -f /etc/rsyslog.d/1-ssplfwd.conf ] ||
 #    cp /opt/seagate/%{product_family}/sspl/low-level/files/etc/rsyslog.d/1-ssplfwd.conf /etc/rsyslog.d/1-ssplfwd.conf
 
-# In case of upgrade start sspl-ll after upgrade
-if [ "$1" == "2" ]; then
-    echo "Restarting sspl-ll service..."
-    systemctl restart sspl-ll.service 2> /dev/null
-fi
-
```

Comment:
Are we not restarting now after `sspl_setup upgrade` so that a new config will be applied? or is provisioned going to do that?

Reply:
Provisioner will start cluster after rpm update. So, I feel its not required

Reply:
yes, rpm will upgraded and then mini-provisioner interface 'upgrade' per component will be executed, where this config upgrade also will also take effect. @nitin-seagate please raise if you see any discrepancy

Just make sure, that old existing config file is not removed or over written when old rpm is uninstalled ( i believe its already taken care, still double confirm)

Reply:
Old config file will not be removed in case of uninstallation